app/views/layouts/_user_menu.html.haml

```diff
@@ -21,7 +21,10 @@
     = link_to(destroy_user_session_path, :method=>'delete') do
       %span.fa.fa-minus
       Sign out
-- if can? :manage, Conference
+- if current_user.is_admin || (current_user.has_any_role? :admin, { :name => :organizer, :resource => :any },
```

Comment:
All of your questions are answered in the ability.rb file. You need to read that file, understand **exactly** what each role can do.  manage the conference and see the admin/conference page are two very different things!  Once you know exactly which abilities each role has, you ll know exactly which ability each role has regarding the Conference model. Once you figure that out, you ll immediately be able to move on!